Example/AppDelegate+CarPlay.swift

```diff
@@ -153,6 +154,44 @@ extension AppDelegate: CarPlayManagerDelegate {
 
 @available(iOS 12.0, *)
 extension AppDelegate: CarPlaySearchControllerDelegate {
+    struct RecentsHolder {
+        static var _recentItems: [RecentItem] = RecentItem.loadDefaults()
+        static var _recentSearchItems: [CPListItem]? = []
+        static var _recentSearchText: String? = ""
+    }
```

Comment:
It looks like this struct is meant solely to group the three properties. I think it would be clearer to move the properties to AppDelegate in order to declare them right next to their respective computed properties, unless we expect to need to reuse this state as a whole in some way.